frontend/pages/admin/UserManagementPage/components/UserForm/UserForm.tsx

```diff
@@ -41,15 +49,17 @@ export interface IFormData {
   name: string;
   newUserType?: NewUserType | null;
   password?: string | null;
-  sso_enabled?: boolean;
+  new_password?: string | null; // if a new password is being set for an existing user, the API expects `new_password` rather than `password`
+  sso_enabled: boolean;
+  two_factor_authentication_enabled?: boolean;
   global_role: UserRole | null;
   teams: ITeam[];
   currentUserId?: number;
   invited_by?: number;
   role?: UserRole;
 }
 
-interface ICreateUserFormProps {
+interface IAddUserFormProps {
```

Comment:
Can we call this `IUserFormProps`, since this is the name of the component and it is used by both `EditUserModal` and `AddUserModal`?